com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -91,8 +91,10 @@ public enum PipelineDebugLevel
     [MovedFrom("UnityEngine.Rendering.LWRP")] public enum RendererType
     {
         Custom,
-        ForwardRenderer,
+        StandardRenderer,
         _2DRenderer,
+        [Obsolete("ForwardRenderer has been renamed StandardRenderer. (UnityUpgradable) -> StandardRenderer", true)]
+        ForwardRenderer,
```

Comment:
Please read  the doc bellow and decide how to proceed with this renaming.

https://confluence.unity3d.com/display/DEV/Member+Reference+Replacement#MemberReferenceReplacement-RENAMING_ENUM_MEMBERSRenamingenummembers

Reply:
Updated constant.